app.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+/*
+I am going to organize my salmon cookie shop's products... create a products' array full of product objects...
+
+Within the product objects
+  cookies
+    cookieName
+    bake_date
+    Price
+    Ingredients
+    Description
+  brownies
+    brownieName
+    Price
+    Ingredients
+    Description
+  smoothies
+    smoothieName
+    Price
+    Ingredients
+    Description
+*/
+
+//
+// lemon.cookieName = 'salmon lemon biscuit';
+// lemon.bake_date ='4/10/17';
+// lemon.price = 4.99;
+// lemon.ingredients = ['salmon','lemons','eggs'];
+// lemon.description = 'yummy lemon and salmon, together!';
+//
+// sugar.cookieName = 'salmon sugar cookie';
+// sugar.bake_date ='4/10/17';
+// sugar.price = 1.99;
+// sugar.ingredients = ['salmon','cinnamon','sugar'];
+// sugar.description = 'delicious salmon and cinnamon sweet cookies, yum!';
+
+//NOTE: I totally missed the assignment domain page at the bottom! starting over!!!!
+
+//1.
+
+
+var FIRSTANDPIKEUL = document.getElementById('first and pike');
+var SEATACAIRPORT = document.getElementById('SeaTac Airport');
+var SEATTLECENTER = document.getElementById('Seattle Center');
+var CAPITOLHILL = document.getElementById('Capitol Hill');
+var ALKI = document.getElementById('Alki');
```

Comment:
Concerning typographical conventions, you should be using camel case rather than all caps.